src/components/themeProvider/__tests__/ThemeProvider.Test.tsx

```diff
@@ -2,7 +2,7 @@ import React, { FC } from 'react';
 import { useTheme } from '@chakra-ui/react';
 
 import { autoScout24Theme, motoScout24Theme } from '../../../themes';
-import { bareRender, screen } from '../../../../.jest';
+import { render, screen } from '../../../../.jest';
```

Comment:
Isn't this using our custom render now?

Reply:
jap mixed that up,and got confused with the naming it's using the render function of the testing lib again